openmedia/gui/controlbox.py

```diff
@@ -101,13 +100,7 @@ def _toggle_shuffle(self, widget):
             player.shuffle = True
 
     def _toggle_playlist(self, widget):
-        if self.playlist_box in self.get_children():
-            self.remove(self.playlist_box)
-            self.get_toplevel().resize(WINDOW_WIDTH, WINDOW_HEIGHT)
-            self.show_all()
-        else:
-            self.pack_start(self.playlist_box, False, False, 0)
-            self.show_all()
+        self.get_toplevel().show_playlist()
```

Comment:
We should signal that the playlist toggle button was pressed rather than assume that our parent is the main window that has a method `show_playlist`. We could fix this in another PR. Could you raise an issue regarding this?